pkg/watcher/reconciler/dynamic/dynamic.go

```diff
@@ -333,22 +414,21 @@ func (r *Reconciler) sendLog(ctx context.Context, o results.Object) error {
 			zap.String("name", o.GetName()),
 		)
 
-		go func() {
```

Comment:
Why don't we pass context here and use select and ctx.done?

Reply:
because the end result is the same; we block the reconcile thread until the operation is complete

no reason for the added complextiy